docs/serving/samples/hello-world/helloworld-java/.dockerignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
Does target will be generated during docker build?

Reply:
this ignores locally generated target/ dir, so we avoid copying unnecessary files to docker daemon during build